include/fmt/chrono.h

```diff
@@ -1518,6 +1578,9 @@ class tm_writer {
 };
 
 struct chrono_format_checker : null_chrono_spec_handler<chrono_format_checker> {
+  bool is_floating_point = false;
+  bool has_precision = false;
```

Comment:
We only need one flag for the condition `has_precision && !is_floating_point`, something like `has_precision_integral`.